explainaboard/loaders/file_loader.py

```diff
@@ -526,6 +527,15 @@ def load_raw(
         self, data: str | DatalabLoaderOption, source: Source
     ) -> FileLoaderReturn:
         config = narrow(DatalabLoaderOption, data)
+
+        # load customized features from global config files
+        customized_features_from_config = get_custmomized_features()
+        if config.dataset in customized_features_from_config.keys():
```

Comment:
```suggestion
        if config.dataset in customized_features_from_config:
```

Reply:
I think `if config.dataset in customized_features_from_config.keys()` is correct here.